src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Technical Specifications
+description: Summary of technical specifications of Stacks 2.0
+---
+
```

Comment:
should we have a section for the STX token here as well? E.g. one partner asked us for the contract that defines our STX token and further token details. Would be nice to have some docs for that (could be another issue/page though)

Reply:
Perhaps the partner assumed STX token is similar to ERC-20 tokens? There isn't a contract that defines STX, it's built into the blockchain. Either way, prefer to address that in a separate PR if needed.